src/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
```

Comment:
I thought it could benefit from having a ".nvmrc" file. Adding this file can greatly help contributors and users who use **Node Version Manager** (**NVM**) to manage Node.js versions for consistency across environments. It ensures that the right Node.js version is used when working on your project, making it more accessible and easier to set up. Consider adding a ".nvmrc" file to specify the supported Node.js version. Thanks for considering this enhancement!